templates/footer.mustache

```diff
@@ -27,11 +27,18 @@
         <div class="col-md-7 contact">
             <h3>{{#str}}stayintouch, theme_moove{{/str}}</h3>
             <h4>{{{getintouchcontent}}}</h4>
+
             <ul>
+                <li><i class="icon-globe"></i><a href="https://www.securitycompass.com" target="_blank">https://www.securitycompass.com</a></li>
```

Comment:
Hey Raja,
Please use `settings.php` to make changes to these fields:

For example:
```
    // Website.
    $name = 'theme_moove/website';
    $title = get_string('website', 'theme_moove');
    $description = get_string('websitedesc', 'theme_moove');
    $default = 'http://conecti.me';
    $setting = new admin_setting_configtext($name, $title, $description, $default);
    $setting->set_updatedcallback('theme_reset_all_caches');
    $page->add($setting);
```

Reply:
Actually, Mitch is correct. We should make the changes in the EN language file, `en/theme_moove.php` instead of settings.php

See:
```
    $description = get_string('bannerheadingdesc', 'theme_moove');
```